ion/tokenizer.go

```diff
@@ -355,6 +361,8 @@ func (t *tokenizer) ReadValue(tok token) (string, error) {
 		str, err = t.readQuotedSymbol()
 	case tokenSymbolOperator, tokenDot:
 		str, err = t.readOperator()
+	case tokenDotSymbol:
+		str, err = t.readDotSymbol()
```

Comment:
I'm somewhat confused by this: a `.` is an operator when it appears in an s-expression. (That is to say: it's a symbol.) Otherwise, it's a plain token.

Shouldn't the `tokenSymbolOperator` and `tokenDot` cases cover this? Are there no other operators that require similar redundancy?

Reply:
Simplified this to no longer use tokenDotSymbol.